### ISSUE TITLE
ErrorDescription problem

### ISSUE THREAD

**Issue by carmas123** [rrousselGit/flutter_hooks#84]
Hi, I try to use your lib but I got always a compilation error: ErrorDescription like this:

```
 Error: The method 'ErrorDescription' isn't defined for the class 'HookElement'.
 - 'HookElement' is from 'package:flutter_hooks/src/framework.dart' ('file:///Users/isigest/Documents/Develop/Tools/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_hooks-0.4.0/lib/src/framework.dart').
Try correcting the name to the name of an existing method, or defining a method named 'ErrorDescription'.
```

**Comment by rrousselGit** [comment]
You must upgrade your Flutter version first

**Comment by carmas123** [comment]
This is my flutter doctor:

```
[✓] Flutter (Channel dev, v1.5.8, on Mac OS X 10.14.4 18E226, locale it-IT)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.2.1)
[✓] Android Studio (version 3.4)
[✓] VS Code (version 1.33.1)
[✓] Connected device (2 available)

• No issues found!

```

**Comment by rrousselGit** [comment]
Hum, you may want to switch to `master` channel temporarily.

For some unknown, they didn't ship the change with the latest release. That's unexpected

**Comment by rrousselGit** [comment]
closes with #85
